<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Devices</class>
 <widget class="QDialog" name="Devices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Devices</string>
  </property>
  <widget class="QPushButton" name="cmdSearch">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOpen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lbDevices">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cmdSearch</tabstop>
  <tabstop>cmdOpen</tabstop>
  <tabstop>cmdCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
